glimr/utils.py

```diff
@@ -219,3 +221,120 @@ def sample_space(space):
         return sample_space(space.sample())
     else:  # non sampleable or nestable value
         return space
+
+
+def top_cv_trials(exp_dir, metric=None, mode="max", model_selection="fold_bests"):
```

Comment:
You'll have to add a docstring to this.

Reply:
Let's discuss this function in person. As expected it's complex. I'd like to understand your thinking and how you group trials based on configuration.